.github/workflows/uats.yaml

```
@@ -24,6 +26,10 @@ jobs:
         run: |
           set -e
 
+          goss_base_url="https://github.com/goss-org/goss/releases/latest/download"
+          curl -L ${goss_base_url}/goss-linux-amd64 -o /usr/local/bin/goss
+          chmod +rx /usr/local/bin/goss
```

Please add a note for replacing this with a snap when available:


```suggestion
          # FIXME: install via the goss snap when available
          goss_base_url="https://github.com/goss-org/goss/releases/latest/download"
          curl -L ${goss_base_url}/goss-linux-amd64 -o /usr/local/bin/goss
          chmod +rx /usr/local/bin/goss
```

---

added comment in fe43de4e381304df0b4ec589b8618f3b3847e783